.github/scripts/clone_engine_repo

```diff
@@ -14,7 +14,7 @@ set -e
 # WARNING! Use "GH" instead of "GITHUB" for variable names to avoid possible
 # conflict with github workflow environment variables.
 
-# Look for a config file and read it if it's there. Intentionally use -x
+# Look for a config file and read it if it's there. Intentionally use -e
```

Comment:
This change now seems to be unnecessary.

Reply:
Actually, this is not a result of merge conflict... There are some problems in clone_engine_repo so I'm thinking to use a different way. I'll handle this in the next PR.